templates/static/scripts.js

```diff
@@ -27,6 +31,18 @@ document.getElementById('url-form').addEventListener('submit', async function(ev
                 headline.setAttribute('data-y', 10); // Set initial y position
                 container.appendChild(headline);
 
+                const cropButton = document.createElement('button');
```

Comment:
How does this button work? I want to be able to drag the edges of the image to crop it.

Reply:
The crop button initializes the Cropper.js library, which allows you to drag the edges of the image to crop it. When you click the "Crop" button, the image is displayed in a cropping interface where you can adjust the crop area by dragging the edges. Once you're satisfied with the crop, you can click the "Crop" button again to apply the crop and update the image.